src/constants.ts

```diff
@@ -140,3 +148,7 @@ export const getUsdcToken = ({
   logoURI:
     "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png",
 });
+
+export const areSettingsVisible = (env: Env): boolean => {
```

Comment:
As we've been talking I would move this function to a different module if we want to keep it.

Reply:
Done!